extensions/git/package.json

```diff
@@ -442,6 +442,11 @@
         "command": "git.stashDrop",
         "title": "%command.stashDrop%",
         "category": "Git"
+      },
+      {
+        "command": "git.openDiff",
```

Comment:
Careful because this will make the command appear in the command palette. It seems it's not meant for that since it requires a `hash` parameter. This isn't really necessary for the command to exist in the system.

Reply:
Yeah, I'll remove it for now. I was planning on eventually having a context menu command attached to the timeline items -- and I can just remove it from the palette once that is there.